src/components/auth/RegistrationForm.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <form
+    class="registration-form"
+    @submit.prevent="register(user)"
+  >
+    <InputCustom
+      label="username"
+      v-model="user.username"
+      :required="true"
+      @input="clearMessage"
+    />
+    <InputCustom
+      label="e-mail"
+      type="email"
+      v-model="user.email"
+      :required="true"
+      @input="clearMessage"
+    />
+    <InputCustom
+      label="password"
+      type="password"
+      v-model="user.password"
+      :required="true"
+    />
+    <InputCustom
+      label="repeat password"
```

Comment:
"repeat" is an action, I think it's better to use here "confirm" like an action but with the clear purpose